src/jdbc/cubrid/jdbc/driver/CUBRIDDriver.java

```diff
@@ -305,36 +304,23 @@ public Connection connect(String url, Properties info) throws SQLException {
         return conn;
     }
 
-    public Connection defaultConnection() throws SQLException {
-        if (UJCIUtil.isServerSide()) {
-            Thread t = Thread.currentThread();
-            Connection c =
-                    (Connection)
-                            UJCIUtil.invoke(
-                                    "com.cubrid.jsp.ExecuteThread",
-                                    "getJdbcConnection",
-                                    null,
-                                    t,
-                                    null);
-            if (c != null) {
-                UJCIUtil.invoke("com.cubrid.jsp.ExecuteThread", "sendCall", null, t, null);
+    public Connection defaultConnection() {
+        try {
+            if (UJCIUtil.isServerSide()) {
+                Thread t = Thread.currentThread();
+                Connection c =
+                        (Connection)
+                                UJCIUtil.invoke(
+                                        "com.cubrid.jsp.ExecuteThread",
+                                        "createConnection",
+                                        null,
+                                        t,
+                                        null);
                 return c;
             }
-
-            UConnection u_con = UJCIManager.connectServerSide();
-            CUBRIDConnection con =
-                    new CUBRIDConnectionDefault(u_con, "jdbc:default:connection:", "default");
-            UJCIUtil.invoke(
-                    "com.cubrid.jsp.ExecuteThread",
-                    "setJdbcConnection",
-                    new Class[] {Connection.class},
-                    t,
-                    new Object[] {con});
-            UJCIUtil.invoke("com.cubrid.jsp.ExecuteThread", "sendCall", null, t, null);
-            return con;
-        } else {
-            return null;
+        } catch (Exception e) {
```

Comment:
Why don't you this exception such as e's stack trace or e.getMessage() ?

Reply:
If defaultConnection() returns null, ExecuteThread in Java SP Server will have a null connection, and the ExecuteThread will log and deal with the exception by throwing NullPointerException.

Reply:
It would be nice to indicate that the code was intentionally left blank.
For example,
/* keep empty */

Reply:
I've added a comment in the catch block

Reply:
It would be better to show the root cause of a problem instead of the NullPointerException